.travis.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     - name: "Release"
       stage: release
-      if: commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
+      if: commit_message ~= /\[trigger release\]/ AND branch ~= /^(ATS-671_skinny_jars_in_modules|master|SP\/.+|HF\/.+)$/
```

Comment:
I assume we revert this change (ie. should not be merged) ?

Reply:
Removed